app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="comment-box">
+<%= form_with model:[@post, @post.comments.build], method: :post, local: true do |form| %>
+  <div >
+    <%= form.text_area :text, size: "80x4", placeholder: "Type your comment", required: true %>
+  </div>
+
+  <div>
+    <%= form.submit "Reply", class: "btn btn-primary", disable_with: "Please wait..." %>
+  </div>
+<% end %>
+</div>
```

Comment:
- [ ] When I create a comment, the post's comments_counter increases by 2, not 1. The same thing when adding a like to a post; the likes_counter increases by 2. Please correct that.